common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -306,7 +306,7 @@ public long maxChunksBeingTransferred() {
    * ceiling of the nearest integer.
    */
   public int chunkFetchHandlerThreads() {
-    if (!this.getModuleName().equalsIgnoreCase("shuffle")) {
+    if (!"shuffle".equalsIgnoreCase(this.getModuleName())) {
```

Comment:
https://github.com/apache/spark/blob/75b396505a266e839b9e2dc1d3ec36eebb6deb43/common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java#L51-L55

Maybe we should change line 55 to `this.module = Objects.requireNonNull(module);` to ensure `this.module` is not null,

Reply:
OK, it's better